### PR TITLE
GA release for NetworkTraversal

### PR DESCRIPTION
**Pull request by AriZavala2**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by AriZavala2** [comment]
Updating Changelog and README in next commit

File: sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/aio/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # --------------------------------------------------------------------------
 
 from ._communication_relay_client_async import CommunicationRelayClient
-from .._generated.models import RouteType
+from .._generated.models import RouteType, CommunicationRelayConfiguration, CommunicationIceServer, CommunicationRelayConfigurationRequest
```
Comment (annatisch) [inline sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/aio/__init__.py:8]: No models should be added to the `aio` public API (including `RouteType`)

File: sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/__init__.py [hunk 2]
```diff
@@ -6,4 +6,4 @@
 
 # pylint: disable=bad-option-value, unused-import
 from ._communication_relay_client import CommunicationRelayClient
-from ._generated.models import RouteType
+from ._generated.models import RouteType, CommunicationRelayConfiguration, CommunicationIceServer, CommunicationRelayConfigurationRequest
```
Comment (annatisch) [inline sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/__init__.py:9]: I believe the `Request` object is internal and shouldn't be added to the public API here.

**Comment by AriZavala2** [comment]
Addressing changes requested in API view. https://apiview.dev/Assemblies/Review/39f6d92be3974ffb92618cea05fb81be @annatisch

File: sdk/communication/azure-communication-networktraversal/CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
 # Release History
 
-## 1.0.0b3 (Unreleased)
+## 1.0.0 (2022-02-03)
 
 ### Features Added
```
Comment (annatisch) [inline sdk/communication/azure-communication-networktraversal/CHANGELOG.md:5]: You can remove the other headings if they're empty :)

File: sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/__init__.py [hunk 4]
```diff
@@ -6,4 +6,4 @@
 
 # pylint: disable=bad-option-value, unused-import
 from ._communication_relay_client import CommunicationRelayClient
-from ._generated.models import RouteType
+from ._generated.models import RouteType, CommunicationRelayConfiguration, CommunicationIceServer
```
Comment (annatisch) [inline sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/__init__.py:9]: I think this is missing a final new line

File: sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/_communication_relay_client.py [hunk 5]
```diff
@@ -37,7 +37,7 @@ class CommunicationRelayClient(object):
     def __init__(
             self,
             endpoint, # type: str
-            credential, # type: TokenCredential
+             credential, # type: TokenCredential
```
Comment (annatisch) [inline sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/_communication_relay_client.py:40]: Looks like an extra space got added?

File: sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/aio/_communication_relay_client_async.py [hunk 6]
```diff
@@ -84,7 +85,7 @@ def from_connection_string(
     async def get_relay_configuration(
             self,
             user: 'CommunicationUserIdentifier' = None,
```
Comment (annatisch) [inline sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/aio/_communication_relay_client_async.py:87]: This method signature is missing the `*` the indicate keyword-only.

**Review by annatisch — APPROVED** [review]

**Pull request merged**